Apps/phawswafv2/awswafv2.json

```
@@ -514,6 +514,167 @@
             },
             "versions": "EQ(*)"
         },
+        {
+            "action": "delete ip set",
+            "identifier": "delete_ip_set",
+            "description": "Remove the specified IP Set",
+            "verbose": "The ip_set_id or ip_set_name must be given as input for deleting an IP set, ip_set_id will be considered if both ip_set_id and ip_set_name is provided in input.",
```

(I think this was the intent with the message, although please ignore/correct it accordingly if not)
```suggestion
            "verbose": "The ip_set_id or ip_set_name must be given as input for deleting an IP set, ip_set_id will be used if both ip_set_id and ip_set_name is provided in input.",
```